### PR TITLE
Operate First services list

### PR DESCRIPTION
**Pull request by bryanmontalvan**
## Description
This commit is based on https://github.com/os-climate/os_c_data_commons/issues/44 which lists OSC services (currently the services are being mocked up and inspired https://odh-dashboard-opf-dashboard.apps.smaug.na.operate-first.cloud/)

## Preview
(Updated: 5/18/2022)
image

### CODE REVIEW

File: src/pages/community-cloud.js [hunk 1]
```diff
@@ -100,14 +101,12 @@ const CommunityCloudPage = () => {
                     </SimpleGrid>
                 </Card>
 
-                <Title order={2} my="md">Workloads</Title>
+                <ODHServices />
```
Comment (durandom) [inline src/pages/community-cloud.js:104]: it's not just ODH services. Let's just call it `Services`

**Review by durandom — COMMENTED** [review]
this is looking great. Do you want to reduce it to the actual services we're running and replace the links with links to the GitOps doc section in https://www.operate-first.cloud/apps/content/README.html ?

**Comment by durandom** [comment]
@bryanmontalvan I would start with a wall of logos, the name and a link of the service. And then point to the entry of the documentation. 
@tumido and others are already working on a proper service catalog which would have more information, like the cluster name or environment.
this page here would only serve as a nice overview on the website.
the more logic we bake into it, the harder to maintain and the more we replicate functionality.

**Comment by HumairAK** [comment]
I think this is great. It may conflicts with the service catalog, but it is a quick solution to a much needed access point for these services (we constantly keep getting questions on how to access x/y services). Let's go with this for now and add the existing services organized by each cluster (we need some way to separate multiple services like JH on smaug from JH on osc). 

Once Service Catalog is active we can replace this if we find it conflicting. 

@bryanmontalvan I have one small request: 

image

Can you remove `services` and replace `osc` with "Popular Services" as the diagram shows? 

Once service catalog is done we can add something like "For a more exhaustive list on the list of services offered, their owners, etc. see service catalog"

File: src/pages/community-cloud.js (hunk 1 above)
Comment (HumairAK) [reply src/pages/community-cloud.js:104]: @bryanmontalvan I agree with this, let's replace mention of odh with just services (including `odh.js`). ODH is one of many services.

File: src/components/nav-tabs/ODH.js [hunk 2]
```diff
@@ -0,0 +1,191 @@
...
+import SupersetLogo from "../../assets/services/superset.svg";
+import PrometheusLogo from "../../assets/services/prometheus.svg";
+import GrafanaLogo from "../../assets/services/grafana.svg";
+import SparkLogo from "../../assets/services/spark.svg";
+import SeldonLogo from "../../assets/services/seldon.svg";
+import KafkaLogo from "../../assets/services/kafka.svg";
+import AirflowLogo from "../../assets/services/airflow.svg";
+import HueLogo from "../../assets/services/hue.svg";
+
+const clusters = [
```
Comment (HumairAK) [inline src/components/nav-tabs/ODH.js:20]: @bryanmontalvan can we get `clusters` this file moved out as it's own `.js` file that we can update ?

**Comment by HumairAK** [comment]
@durandom can you see why the netfliy integration is not running? was the cap reached here?

**Comment by durandom** [comment]
netlify just got stuck. works now.

@bryanmontalvan can you remove the dropdown? The user should not care about the cluster. Let's just put all services on one grid

**Comment by bryanmontalvan** [comment]
The latest update:
image

Looking for some feedback/descriptions for each service :)

File: src/components/ServiceList.js [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+import ObservatoriumLogo from "../assets/services/observatorium.svg";
+import RedHatLogo from "../assets/services/redhat.svg";
+import Dexlogo from "../assets/services/dex.svg"
+
+const services = [
+    {
+        title: 'Grafana',
+        image: GrafanaLogo,
+        url: 'https://grafana.com/docs/grafana/latest/',
+        description: 'Visualization and analytics software, which we use for monitoring and dashboards',
```
Comment (durandom) [inline src/components/ServiceList.js:14]: Get the punchline from Wikipedia: https://en.wikipedia.org/wiki/Grafana
or the project page: https://grafana.com/grafana/

In this case I would opt for: `Dashboard anything. Observe everything.`

**Review by durandom — COMMENTED** [review]
as for the example of grafana. 
* the link should be to our docs. If we don't have docs, don't put it on the page
* the blurb should be super generic, from the projects page. No need to re-invent marketing :)

File: src/components/nav-tabs/Services.js [hunk 4]
```diff
@@ -0,0 +1,93 @@
...
+    return (
+        <main>
+            <Container pb={69} style={{paddingLeft: 0, paddingRight: 0}}>
+                <Title order={2} my="md">Our Services</Title>
+                <Card withBorder radius="md" className={classes.card}>
+                    <Group position="apart">
+                        <Anchor size="xs" color="dimmed" sx={{ lineHeight: 1 }}>
+                            ...
+                        </Anchor>
+                    </Group>
```
Comment (tumido) [inline src/components/nav-tabs/Services.js:83]: What is the purpose of this? Do we need this? It does nothing on click.

Comment (bryanmontalvan) [reply src/components/nav-tabs/Services.js:83]: We do not! Not sure how that slipped under my belt. I was planning on using that as a filter dropdown

**Review by tumido — APPROVED** [review]
/lgtm

Looks great! :heart:

**Review by durandom — CHANGES_REQUESTED** [review]
@bryanmontalvan can you replace the links for all the services to point to our documentation at https://www.operate-first.cloud/apps/content/README.html instead of the generic homepages? The grafana link is good

**Comment by bryanmontalvan** [comment]
@durandom Done, I could not find the respective pages for ODF and Dex OIDC so I kept their links as is :+1:

**Review by schwesig — COMMENTED** [review]
/lgtm

**Review by durandom — APPROVED** [review]
@bryanmontalvan thanks a lot. this looks good 😍 

/approve
/lgtm

**Pull request merged**